concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-92	92	a fish, Leuciscus souffia agassizi	227	FISH
+RonaTas-2011-431-93	93	only, just	1580	ONLY
+RonaTas-2011-431-94	94	pick-axe; mace; a small axe; war hammer		
+RonaTas-2011-431-95	95	nettle, Urtica	33	NETTLE
+RonaTas-2011-431-96	96	thicket, brushwood, dry foliage, grove, meadow	2819	GRASSLAND
+RonaTas-2011-431-97	97	thicket, brushwood; dry foliage, grove, meadow	2819	GRASSLAND
+RonaTas-2011-431-98	98	(silver) cup, wooden ladle, earthenware bowl	476	CUP
+RonaTas-2011-431-99	99	fighting, troop, army group, battle, quarrel, hubbub, flock	1848	QUARREL
+RonaTas-2011-431-100	100	to add, to join; to buckle up something	1743	JOIN
+RonaTas-2011-431-101	101	clasp		
+RonaTas-2011-431-102	102	measle, smallpox, pocks; skin eruption; decorative embroidery	1054	SMALLPOX
+RonaTas-2011-431-103	103	to bind (something)	1917	TIE
+RonaTas-2011-431-104	104	impurities, tow, chaff	2363	CHAFF (HUSK)
+RonaTas-2011-431-105	105	snail, Helix; snail-shell, pulley, vertebra	1543	SNAIL
+RonaTas-2011-431-106	106	a wine of low quality, a fruit wine		
+RonaTas-2011-431-107	107	secretion, mucus discharged from the eyes		
+RonaTas-2011-431-108	108	squint-eyed, blind in one eye, onearmed, one-legged	87	SQUINT-EYED
+RonaTas-2011-431-109	109	a carpenters axe with a long helve		
+RonaTas-2011-431-110	110	stallion	936	STALLION
+RonaTas-2011-431-111	111	sexual organ of large male animals, of a bull; a whip made from it		
+RonaTas-2011-431-112	112	feast, christening feast (at baptism)		
+RonaTas-2011-431-113	113	to become smaller, remain small, to sway, to totter, tosprain	573	SWAY
+RonaTas-2011-431-114	114	stubborn	3634	STUBBORN
+RonaTas-2011-431-115	115	field-guard; herald; Turkish envoy		
+RonaTas-2011-431-116	116	rough, coarse; ugly; wild, unbridled	1923	ROUGH
+RonaTas-2011-431-117	117	to wind, to turn, to distrort, misinterpret, to spin a yarn	1588	TURN
+RonaTas-2011-431-118	118	bobbin, spindle	277	SPINDLE
+RonaTas-2011-431-119	119	excellent; worthless		
+RonaTas-2011-431-120	120	a short fur coat		
+RonaTas-2011-431-121	121	grist, groats, soft hail		
+RonaTas-2011-431-122	122	noon, south	12	MIDDAY
+RonaTas-2011-431-123	123	nut; walnut, Juglans	2042	WALNUT
+RonaTas-2011-431-124	124	pig, Sus domestica	1337	PIG
+RonaTas-2011-431-125	125	to lean, topple over, stream down	3619	LEAN
+RonaTas-2011-431-126	126	to squeeze, to thrust into	1414	SQUEEZE
+RonaTas-2011-431-127	127	church (building and organisation)	1349	CHURCH
+RonaTas-2011-431-128	128	wedge		
+RonaTas-2011-431-129	129	plough	2154	PLOUGH (INSTRUMENT)
+RonaTas-2011-431-130	130	to allow, permit, concede; yield, give way	3238	ALLOW
+RonaTas-2011-431-131	131	to propitiate		
+RonaTas-2011-431-132	132	community work in agriculture, a kind of corvée		
+RonaTas-2011-431-133	133	to arrive, to reach, get to	1387	ARRIVE
+RonaTas-2011-431-134	134	merit		
+RonaTas-2011-431-135	135	morals, morality		
+RonaTas-2011-431-136	136	to tire, lose vitality, slacken, relax	168	REST
```

Comment:
The main meaning seems to be `1996 GET TIRED`, I'd change the mapping or unmap due to the presence of "relax"

Reply:
I'd also unmap.

Reply:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534